rclpy/rclpy/__init__.py

```diff
@@ -181,7 +184,11 @@ def create_node(
         ))
 
 
-def spin_once(node: 'Node', *, executor: 'Executor' = None, timeout_sec: float = None) -> None:
+def spin_once(
+        node: 'Node',
+        *,
+        executor: Optional[Executor] = None,
```

Comment:
I'm pretty sure this change (and the related ones below) are what are causing CI to fail.  Maybe:

```suggestion
        executor: Optional['Executor'] = None,
```

will work?